ui/types.go

```diff
@@ -0,0 +1,14 @@
+package ui
+
+// TextOpts Asking for text options
+type TextOpts struct {
+	Label        string
+	DefaultValue string
+}
+
+// ChoiceOpts asking for choice options
+type ChoiceOpts struct {
+	Label        string
+	DefaultValue int
```

Comment:
I think it should be a string as user will not remember at which index his default value will be present.